lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -37,12 +37,15 @@ sub dbh {
     else {
         my $database = $self->config->MYSQL_database;
         my $host     = $self->config->MYSQL_host;
-        my $user     = $self->config->MYSQL_user();
-        my $password = $self->config->MYSQL_password();
+        my $port     = $self->config->MYSQL_port;
+        my $user     = $self->config->MYSQL_user;
+        my $password = $self->config->MYSQL_password;
+
+        my $data_source_name = "DBI:mysql:database=$database;host=$host;port=$port";
```

Comment:
Is there any risk of warning from DBI if port is set when host is 'localhost'?

Reply:
Do you mean, does the DBI emits warning in this case ? Not that I am aware of.

Reply:
We should test for that so that we do not get unwanted warnings.

Reply:
If DBD::mysql starts emitting warnings those will be caught in config.t. It's already using Test::NoWarnings.

Reply:
Well, I cannot see any message in the daemons' logs. Should I look somewhere else to see if warning are raised by the driver in this case ?

Reply:
Then it should be fine.